src/SourceLocator/Reflection/SourceStubber.php

```diff
@@ -64,11 +64,20 @@ public function __invoke(CoreReflectionClass $classReflection) : string
     {
         $classNode = $this->createClass($classReflection);
 
-        $this->addClassModifiers($classNode, $classReflection);
+        if ($classNode instanceof Class_) {
+            $this->addClassModifiers($classNode, $classReflection);
```

Comment:
`Interface_` and `Trait_` should be passed here too

Reply:
How can we have protected/private interfaces or traits?

Reply:
No fix needed